docs/cloud-adoption/governance/governance-multiple-teams.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: "Governance design guide: new development in Azure for multiple teams"
+title: "Enterprise Cloud Adoption: Governance design in Azure for multiple teams"
 description: Guidance for configuring Azure governance controls for multiple teams, multiple workloads, and multiple environments
-author: petertay
+author: petertaylor9999
 ---
 
-# Governance design guide: new development in Azure for multiple teams
+# Enterprise Cloud Adoption: Governance design for multiple teams
 
-The goal of this guidance is to help you learn the process for designing a resource governance model in Azure.  We'll look at a set of hypothetical governance requirements, then go through several example implementations that satisfy those requirements. 
+The goal of this guidance is to help you learn the process for designing a resource governance model in Azure to support multiple teams, multiple workloads, and multiple environments.  We'll look at a set of hypothetical governance requirements, then go through several example implementations that satisfy those requirements. 
 
 Our requirements are:
 * Identity management for multiple teams with different resource access requirements in Azure. The identity management system stores the identity of the following users:
```

Comment:
In the section "Resource management model," some subheadings would be useful (H3's?)  because there's a lot of moving parts to keep in your head.

Maybe add a subhead here:

> Let's begin by evaluating the first option.

And another for the 2d option

Reply:
This whole section is going to be moved into the "getting started" section as its own restructured doc in the next iteration. So, let's leave this as is until then.